### PR TITLE
Add Chinese language

### PR DESCRIPTION
**Pull request by SuperYYT**
i add chinese lang into this plugins，would you adopt it~？
tips：welcome to my website ：[www.minebbs.com](url) MineBBS

### CODE REVIEW

**Comment by fuyutsuki** [comment]
Thank you for your PR!

P.S. I knew about MineBBS from before, but I'm learning Chinese so it is hard to read 🤣

File: resources/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Texter config file
 
-# set language (en_us, ja_jp or ru_ru)
-locale: en_us
+# set language (zh_cn, en_us, ja_jp or ru_ru)
+locale: zh_cn
```
Comment (fuyutsuki) [inline resources/config.yml]: English is the official language, so do not change the default

please change as below
```yaml
# set language (en_us, ja_jp, ru_ru or zh_cn)
locale: en_us
```

File: resources/language/zh_cn.ini [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+command.txt.add.success=FloatingText: {%0} 已创建
+command.txt.edit=/txt e(dit)
+command.txt.edit.success=FloatingText: 在 {%0} 上编辑 {%1}
+command.txt.move=/txt m(ove)
+command.txt.move.success=FloatingText: 将 {%0} 移动至 {%1}
+command.txt.remove=/txt r(emove)
+command.txt.remove.success=FloatingText: 删除 {%0}
+
+form.ftname=浮动文本名称
+form.ftname.unique=浮动文本名称(unique)
```
Comment (fuyutsuki) [inline resources/language/zh_cn.ini]: Does `unique` mean to Chinese people?

File: resources/config.yml (hunk 1 above)
Comment (SuperYYT) [reply resources/config.yml]: ok

File: resources/language/zh_cn.ini (hunk 2 above)
Comment (fuyutsuki) [reply resources/language/zh_cn.ini]: I think `unique` is `唯一`

Comment (SuperYYT) [reply resources/language/zh_cn.ini]: Thank you, i don't understand it when i translate it.

Comment (SuperYYT) [reply resources/language/zh_cn.ini]: I'll change it.

**Comment by SuperYYT** [comment]
> Thank you for your PR!
> 
> P.S. I knew about MineBBS from before, but I'm learning Chinese so it is hard to read 🤣

Yes, chinese is a little difficult to learn😄, I wish you could learn it. 
We also offer multiple languages button at the bottom of website.
（But it can't translate the content of the thread, you may ues google translate）
image

**Comment by fuyutsuki** [comment]
Thank you for your cooperation!
I will release it on v3.4.3 (next patch update)!

and going to register MineBBS 😄

**Pull request merged**